relayer/relays/ethereum/parachain-writer.go

```diff
@@ -53,7 +55,7 @@ func (wr *ParachainWriter) Start(ctx context.Context, eg *errgroup.Group) error
 	}
 	wr.genesisHash = genesisHash
 
-	wr.pool = parachain.NewExtrinsicPool(eg, wr.conn)
+	wr.pool = parachain.NewExtrinsicPool(eg, wr.conn, 10)
```

Comment:
Does this need to use the constant `MaxWatchedExtrinsics`? And can we set to a larger value such as 32? I think by default the ethereum relayer submits 12 headers at a time
```suggestion
	wr.pool = parachain.NewExtrinsicPool(eg, wr.conn, MaxWatchedExtrinsics)
```

Reply:
@alistair-singh I'll add the config! 😄

Whoops, you're right, it should be using the constant.

I left the old Ethereum relayer with MaxWatchedExtrinsics = `10` (as it was), but the Beacon relayer [submits `32` header at a time](https://github.com/Snowfork/snowbridge/pull/688/files#diff-8788c1408defd89dcb3fb8a2b398665badf8ef0fe2abe6451a7b0bad9aafec6aR14).